tests/bluetooth/tester/src/audio/btp_cap.c

```diff
@@ -450,11 +450,17 @@ static uint8_t btp_cap_unicast_audio_stop(const void *cmd, uint16_t cmd_len,
 	return BTP_STATUS_SUCCESS;
 }
 
-static struct bt_cap_initiator_broadcast_subgroup_param
-	cap_subgroup_params[CONFIG_BT_BAP_BROADCAST_SRC_SUBGROUP_COUNT];
-static struct bt_cap_initiator_broadcast_stream_param
-	cap_stream_params[CONFIG_BT_BAP_BROADCAST_SRC_SUBGROUP_COUNT]
-		     [CONFIG_BT_BAP_BROADCAST_SRC_STREAM_COUNT];
+struct bt_cap_initiator_broadcast_params {
```

Comment:
```suggestion
struct cap_initiator_broadcast_params {
```

Please avoid using the `bt_` prefix for non-stack defined values